omaha/tools/retry.py

```diff
@@ -15,6 +18,16 @@
     print 'Retrying %d...' % i
   retcode = subprocess.call(cmd)
   if retcode == 0:
+    # [Sparrow]
+    # Only a viasat machine should have this essential file.
+    if (os.path.isfile(r'C:\Git\sbb\scripts\slave\sign_viasat_client_files.py') and 
+        str(sys.argv[3]).endswith('signtool.exe') and 
+        str(sys.argv[-1]).split('.')[-1] in ['exe', 'msi']):
+      ret_code = subprocess.call([r'C:\Python27\python.exe', 
+                                 r'C:\Git\sbb\scripts\slave\sign_viasat_client_files.py', 
```

Comment:
I'd like to suggest to use environment variables for setting up an absolute path

Reply:
The issue with environment variables is that this retry.py script isn't run in the same environment. It uses the base python environment without any of the environment variables that we've set. This is part of the reason I moved the signing code out of the Omaha Client code.